pkg/views/gitprovider/select.go

```diff
@@ -125,6 +128,67 @@ func GitProviderSelectionView(gitProviderAddView *apiclient.SetGitProviderConfig
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	var selectedSigningMethod string
+
+	commitsigningDataForm := huh.NewForm(huh.NewGroup(
+		huh.NewSelect[string]().
+			Title("Commit Signing Method").
+			Options(
+				huh.Option[string]{Key: "None", Value: "none"},
+				huh.Option[string]{Key: "SSH", Value: "ssh"},
+				huh.Option[string]{Key: "GPG", Value: "gpg"},
+			).
+			Value(&selectedSigningMethod).WithHeight(6),
+	).WithHeight(5).WithHideFunc(func() bool {
+		return isDeleting || providerNotSupportCommitSigning(gitProviderAddView.Id)
+	}),
+		huh.NewGroup(
+			huh.NewInput().
+				Title("Signing Key").
+				Value(gitProviderAddView.SigningKey).
```

Comment:
You should add another helper variable for inputing the signing key because this pointer should not be set if the value is `none`.

This is what was causing the issue in your gif a few comments ago

Reply:
done